NAMESPACE

```diff
@@ -98,7 +99,15 @@ exportMethods(colnames)
 exportMethods(getMart)
 exportMethods(setMart)
 exportMethods(show)
+importFrom(BiocGenerics,"strand<-")
 importFrom(BiocGenerics,colnames)
+importFrom(BiocGenerics,strand)
+importFrom(GenomeInfoDb,seqlengths)
+importFrom(GenomicRanges,"strand<-")
+importFrom(GenomicRanges,GRanges)
+importFrom(GenomicRanges,strand)
+importFrom(IRanges,IRanges)
+importFrom(S4Vectors,mcols)
```

Comment:
Why are we doing this?
I suspect that the function just needed :: adding to the call.